missions/FOM_Missions.txt

```diff
@@ -1812,41 +2538,124 @@ fom_missions_fith_slot = {
 		position = 7
 		
 		provinces_to_highlight = {
-			OR = {
-				area = Eientei_state
-				area = Bamboo_forest_of_the_lost
-				area = Fujiwara_state
-			}
+			region = bamboo_forest_region
 			NOT = { country_or_non_sovereign_subject_holds = ROOT }
 		}
 		
 		trigger = {
-			custom_trigger_tooltip = {
-				tooltip = FOM_expand_the_ingredient_sources.tooltip
+			
+			
 				num_of_provinces_owned_or_owned_by_non_sovereign_subjects_with = {
-					value = 6
-					OR = {
-						area = Eientei_state
-						area = Bamboo_forest_of_the_lost
-						area = Fujiwara_state
-					}
+					value = 60 # 84 total
+					region = bamboo_forest_region
 				}
-			}
+			
 		}
```

Comment:
Some indentation work needed here, also not sure why the decision to remove the custom trigger tooltip.

Reply:
The custom trigger did nothing special and was unnecessary